react/components/Popover.js

```diff
@@ -47,7 +53,9 @@ export default class Popover extends Component {
   render() {
     const { renderHeader, children } = this.props
 
-    const boxPositionStyle = {
+    const isMobile = path(['__RUNTIME__', 'hints', 'mobile'], global)
```

Comment:
use withContextRuntime hoc to get the contextProps of runtime. See example in Search-result and rebuy apps